websites/G/galaxy/presence.ts

```diff
@@ -0,0 +1,462 @@
+/* eslint-disable no-one-time-vars/no-one-time-vars */
```

Comment:
Do not needlessly disable eslint rules
```suggestion
```

Reply:
:+1:, I just thought that in some areas having variables would make the code slightly easier to read, especially in string interpolation.